src/Command/InfectionCommand.php

```diff
@@ -274,8 +274,9 @@ private function includeUserBootstrap(InfectionConfig $config): void
 
     private function applyMemoryLimitFromPhpUnitProcess(Process $process, MemoryUsageAware $adapter): void
     {
-        if (\PHP_SAPI === 'phpdbg') {
+        if (\PHP_SAPI === 'phpdbg' || \extension_loaded('xdebug')) {
```

Comment:
So with this change we will change memory limit only when we use `--coverage` option. Iis that what @sanmai wanted to achieve when he developed that part?

Reply:
Normally you're not allowed to change the system php.ini (there are checks to taht). Normally means not running Infection under a root user.

Reply:
If xdebug handler is active, then xdebug won't be active at this point of the application.

But as per https://github.com/infection/infection/pull/443#issuecomment-412418591, there may be better ways of doind this check